jest.config.js

```diff
@@ -1,8 +1,3 @@
 module.exports = {
   collectCoverageFrom: ['lib/**/*.js'],
-  // Work-around for subpath imports.
-  // See: https://giters.com/facebook/jest/issues/11100
-  "moduleNameMapper": {
-    "@eslint/eslintrc": "@eslint/eslintrc/dist/eslintrc-universal.cjs"
```

Comment:
lol, that was it?

Reply:
I'm honestly not sure what this was I looked at the link in the comments but 🤷‍♂️ . I think it was some sort of workaround apparently. Upgrading Jest with this config caused errors... removing it then fixed them. LOL